src/components/crown/crownConfig/crownConfig.vue

```diff
@@ -222,8 +222,6 @@ export default {
       if (!store.getters.allowSavingElementPosition) {
         return;
       }
-
-      this.$emit('save-state');
```

Comment:
is it necessary to remove this $emit? It looks like the undo for actions started from the crow can be affected. Can you explain?